packages/api-plugin-fulfillment/src/index.js

```diff
@@ -0,0 +1,57 @@
+import { createRequire } from "module";
+import mutations from "./mutations/index.js";
+import queries from "./queries/index.js";
+import resolvers from "./resolvers/index.js";
+import schemas from "./schemas/index.js";
+import startup from "./startup.js";
+import preStartup from "./preStartup.js";
+import { MethodEmptyData } from "./simpleSchemas.js";
+import { fulfillment, registerPluginHandlerForFulfillmentTypes } from "./registration.js";
+
+const require = createRequire(import.meta.url);
+const pkg = require("../package.json");
+
+/**
+ * @summary Import and call this function to add this plugin to your API.
+ * @param {Object} app The ReactionAPI instance
+ * @returns {undefined}
+ */
+export default async function register(app) {
+  await app.registerPlugin({
+    label: "Fulfillment",
+    name: "fulfillment",
+    version: pkg.version,
+    collections: {
+      FulfillmentRestrictions: {
+        name: "FulfillmentRestrictions",
+        indexes: [
+          [{ methodIds: 1 }]
+        ]
+      },
+      Fulfillment: {
+        name: "Fulfillment",
```

Comment:
Any reason not to name this collection `FulfillmentTypes` everywhere? The schema is named that and there are arguments such as `fulfillmentTypeId`, so it would make sense for the collection name to match.

Reply:
I vaguely remember there was an initial discussion with @zenweasel on this where I changed the name from Fulfillments to Fulfillment.

What I remember is that a collection which had the FulfillmentTypes and also its FulfillmentMethods, was called Fulfillment. Not sure if that is a strong enough reason. 

If we have to make this change, I guess we have to update most of the mutations/queries/resolvers.

Reply:
I don't have a strong opinion here, I think the relevant decider is "element of least surprise" which I _think_ might lean towards `Fulfillment` since it covers everything about Fulfillment, not just fulfillment types similar to the way "Shipping" did